resources/sync_gateway_configs/sync_gateway_default_functional_tests_no_port_di.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do we want to include couchbase in the filename?

Reply:
same as above